CMakeLists.txt

```diff
@@ -3,10 +3,13 @@ project(nix_mx CXX)
 
 set(VERSION_MAJOR 1)
 set(VERSION_MINOR 4)
-set(VERSION_PATCH 0)
+set(VERSION_PATCH 2)
 
 set(VERSION_ABI   1)
 
+ENABLE_LANGUAGE(C)
+cmake_policy(SET CMP0050 NEW)
```

Comment:
I comment here would be useful

Reply:
haha, you found the weak point. I got an cmake error that language c was not enabled, so I did this. The cmake_policy warning could be dealt with by explicitly stating the policy. The policy changes do not apply to our cmake solution, afaik, so I chose it... If you have more insight into it, however, ...